src/components/ClockModule.js

```diff
@@ -0,0 +1,281 @@
+          <div
+            css={`
+              display: flex;
+              align-items: center;
+              justify-content: space-between;
+            `}
+          >
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+              `}
+            >
+              <img
+                alt=""
+                width={28}
+                height={28}
+                src={logoSvg}
+                css={`
+                  margin-right: ${0.5 * GU}px;
+                `}
+              />
+              <span>Aragon Court</span>
+            </div>
+            <div
+              css={`
+                display: flex;
+                align-items: center;
+              `}
+            >
+              <span
+                css={`
+                  margin-right: ${0.5 * GU}px;
+                `}
+              >
+                {shortenAddress(courtConfig?.id)}
+              </span>
+              <ButtonBase
+                disabled={!courtConfig}
+                href={blockExplorerUrl('address', courtConfig?.id, {
+                  networkType,
+                })}
+              >
+                <IconExternal
+                  css={`
+                    color: ${theme.contentSecondary};
+                  `}
+                />
+              </ButtonBase>
+            </div>
+          </div>
+          <div
+            css={`
+              color: ${isSynced ? theme.positive : theme.negative};
+              display: flex;
+              align-items: center;
+              margin-top: ${1 * GU}px;
+            `}
+          >
+            <IconSync />
+            <span
+              css={`
+                ${textStyle('body4')}
+                text-transform: uppercase;
+              `}
+            >
+              {isSynced ? 'Term is in sync' : 'Term is out of sync'}
+            </span>
+          </div>
+          <div
+            css={`
+              margin-top: ${1 * GU}px;
+            `}
+          >
+            <span>
+              <span
+                css={`
+                  color: ${theme.contentSecondary};
+                  margin-right: ${0.5 * GU}px;
+                `}
+              >
+                {isSynced ? 'Synced' : 'Out of sync'}:
+              </span>
+              {isSynced ? 'Current' : 'Last'} term {currentTermId}
+            </span>
+          </div>
+          <div
+            css={`
+              margin-top: ${1.5 * GU}px;
+            `}
+          >
+            {isSynced ? (
+              <span
+                css={`
+                  ${textStyle('body3')};
+                  color: ${theme.contentSecondary};
+                `}
+              >
+                Term is the time unit for measuring Aragon Court phases. Each
+                term lasts {formatDuration(courtConfig.termDuration / 1000)}
+              </span>
+            ) : (
+              <Button
+                label="Call Heartbeat"
```

Comment:
Not sure if “heartbeat” is a name or a noun here, but if it’s the latter we shouldn’t capitalize it:

```suggestion
                label="Call heartbeat"
```

Reply:
I think in this case is intended to be a name? cc. @dizzypaty

Reply:
From an offline chat with @dizzypaty, we are rewording most labels including the `Call heartbeat ` to `Update term`. Here is the commit https://github.com/aragon/court-dashboard/pull/227/commits/e80c35b1e5454cfa489dceaba8fdb4d31a08b08e